README.md

```diff
@@ -95,6 +95,8 @@ This role will install the latest version of Jenkins by default (using the offic
     jenkins_repo_url: deb http://pkg.jenkins-ci.org/debian-stable binary/
     jenkins_repo_key_url: http://pkg.jenkins-ci.org/debian-stable/jenkins-ci.org.key
 
+You can conveniently change the defaults irrespective of the platform by setting `jenkins_prefer_lts: true`.
```

Comment:
Maybe to make the whole thing simpler, we could just remove the examples above (in the documentation), and only document the `jenkins_prefer_lts` variable. In its description, we can mention "if you need to change the jenkins repo details, you can also manually override the `jenkins_repo_url` and `jenkins_repo_key_url`?

Reply:
Yea, I think you're right.

Reply:
Since the value of `jenkins_repo_url` on Debian/Ubuntu is more than just a URL, but rather a "source string", I think we're better off keeping a good example in the documentation. However, I've updated the docs to direct users to prefer the new `jekins_prefer_lts` to reduce their configuration overhead when official repositories are acceptable.